### ISSUE TITLE
[Bug]: mulible results with same model name not working

### ISSUE THREAD

**Issue by Git-Maniac** [BlafKing/sd-civitai-browser-plus#170]
### Describe the bug.

if you search a model and get more than one results with the same model name, you can only download the 1st one. All other results download the 1st one again

### Steps to reproduce the problem.

1. Search model name like "chastain SDXL"
2. select the 1st result, you can download it
3. select the 2nd result, it download the 1st one again

### Expected behavior

Download the files from the selected model

### System info

* Extension version: 3.2.4 
* OS: Windows 10 x64 22H2
* SD-WebUI version: 1.7.0
* Python: .3.10.11

### Console logs

```Shell
No error just wrong files downloaded
```

### Additional information

As workaoround you can search by the user name, therefore you never have results with same model name

**Comment by BlafKing** [comment]
Thanks for reporting!

The model selection was done by model name, instead of something which is always unique like a model ID.

Was way more work than I anticipated, since a lot of the code was built on detecting models by their name, but I have now switched to selecting models by their ID instead in ([v3.2.5](https://github.com/BlafKing/sd-civitai-browser-plus/commit/c3129a7fd3d4cc63378027725e8fc67de15fdedb))!

This bug should now be fixed so I'll close this issue! :)
